Client/Frontend/Browser/TabCell.swift

```diff
@@ -168,16 +161,18 @@ class TabCell: UICollectionViewCell, TabTrayCell, ReusableCell {
     }
 
     // MARK: - Configure tab cell with a Tab
-    func configureWith(tab: Tab, isSelected selected: Bool) {
+    func configureWith(tab: Tab, isSelected selected: Bool, theme: Theme) {
         isSelectedTab = selected
 
+        applyColors(theme)
```

Comment:
most minor of nits: `applyColors(from: theme)` reads nicer. That having been said, I'm in NO way worried about it.

Reply:
I also see this pattern everywhere else, so, that makes this an even more minor nit, maybe.

Reply:
Is this a naming convention we are pushing for throughout the app? I personally don't like this naming convention because I've never seen a good set of rules for when and how to apply it and it looks so much worse when poorly used. That being said I'm fine with switching to this pattern if it is or is going to be a part of our style guide.

Reply:
We are absolutely not pushing for that. As for style guide, that's something we should discuss when it comes up, but for now, no changes are needed, which is why I just mentioning it as a very minor nit for myself.